### PR TITLE
Update Gradle plugin link to another plugin.

### PR DESCRIPTION
**Pull request by eshepelyuk**
Hello

The [plugin currently mentioned](https://github.com/alphagov/gradle-gatling-plugin) has several disadvantages

1. It is not published neither in _Maven Central_ nor in _Gradle Plugin Repo_
1. The only way to use it - is to download sources and build locally, that makes usage experience extremely unfriendly.
1. Documentation is basic and there's no any configuration options available.
1. No tests.
1. There were no activity on this plugin in more than a year (last commit is dated by16th of February, 2015) although there's 4 issues are opened.

Basically, I am not sure that this plugin works at all.

From the other hand, there's [plugin of another author](https://github.com/lkishalmi/gradle-gatling-plugin) available on `GitHub`.

The second plugin is alive and new version was released recently with my contributions, so currently it provides following features.

1. Availability in `Gradle Plugin Repo`, so installation of it is a piece of cake just by adding line into build script.
1. Comprehensive documentation with usage examples.
1. Configurations options are available to extend flexibility of usage.
1. Possibility to run single `simulation` as well as multiple _simulations_ at once.
1. Advanced IDE integration (verified in IDEA and NetBeans)
1. Seamless integration of `Gatling` simulations into working project, separating `Gatling` sources from production code.
1. Functional test coverage.

So, the purpose of this small PR is to update `Gradle` plugin link in `Gatling` docs to [@lkishalmi version](https://github.com/lkishalmi/gradle-gatling-plugin).

Awaiting for your feedback.

### CODE REVIEW

File: src/sphinx/extensions/index.rst [hunk 1]
```diff
@@ -17,7 +17,7 @@ Official Extensions
 Third Parties
 =============
 
-* `Gradle plugin <https://github.com/alphagov/gradle-gatling-plugin>`_
+* `Gradle plugin <https://github.com/lkishalmi/gradle-gatling-plugin>`_
```
Comment (slandelle) [inline src/sphinx/extensions/index.rst]: Please don't replace. This plugin seems to have been working for most people for quite some time and project seems to be still active.

Just add yours and add the author name in each link.

Comment (slandelle) [inline src/sphinx/extensions/index.rst]: Forget my previous comment. It seems @alxndrsn got a new hair style, moved to a different country and is not longer involved. Damn, the guy owes a dinner!

**Comment by eshepelyuk** [comment]
Thank you for accepting the contribution. 

May I ask - when this going to be available on `Gatling` site? Only after release of `2.2.0`

**Comment by slandelle** [comment]
> Only after release of 2.2.0

Yes. We're working like crazy, but should be pretty soon.

**Comment by eshepelyuk** [comment]
Great news, thanks for your work.

File: src/sphinx/extensions/index.rst (hunk 1 above)
Comment (alxndrsn) [inline src/sphinx/extensions/index.rst]: Ouch, it's all true!  :+1: for an actively supported gradle plugin.  Now how can I repay that dinner?

Comment (slandelle) [inline src/sphinx/extensions/index.rst]: Hey, you're alive! Your github profile says Barcelona, are you still there? I think the one you owe is @ceeaspb, who's in London.

Comment (alxndrsn) [inline src/sphinx/extensions/index.rst]: Yep, in Barcelona so not sure what alphagov are doing with gatling these days.  @ceeaspb is it you?

**Pull request merged**